windows-installer/README.md

```diff
@@ -0,0 +1,92 @@
+Windows installer script
+========================
+
+Overview
+--------
+
+The windows installer script found in this directory builds a windows installer
+executable capable of installing both 32 and 64 bit openssl binaries, along
+with their corresponding development headers
+
+Requirements
+------------
+
+* [NSIS](https://nsis.sourceforge.io/Main_Page) version 3.0.8 or later
+* Windows 2022 or later
+* The Windows SDK
+  - The makecert.exe utility (to demonstrate installer signing)
+  - The Pvk2Pfx.exe utility (to demonstrate installer signing)
+  - The SignTool.exe utility (to demonstrate installer signing)
+
+Notes on Signing
+----------------
+
+Installer signing is demonstrated here using self signed certificates. Do not
+use this signed code in a deployment as the self signed certificate should not
+be trusted.  However, if you wish to observe this signed installer in
+operation, the self signed certificate may be imported to the local trust store
+following the instructions
+[here](https://learn.microsoft.com/en-us/windows/win32/appxpkg/how-to-create-a-package-signing-certificate).
+Note: Importing this example certificate to the trust store is done at your own
+risk
+
+Installer Build Prerequisites
+-----------------------------
+
+1) Build Openssl from the parent of this directory:
+    a) clone the openssl repository
+    b) cd path\to\openssl\
+    c) mkdir \_build64
+    d) cd \_build64
+    e) perl ..\Configure [options] VC-WIN64A
+    f) nmake
+    g) nmake DESTDIR=..\install64
+    h) repeat steps b-g substituting 32 for 64 to build VC-WIN32

```

Comment:
Apparently this does not format well in https://github.com/openssl/installer/blob/58d08724a7478691631fd7f829d87aca39ca3e38/windows-installer/README.md

Perhaps just use code format?